docs/modding/suitmodding/addingblobs.md

```diff
@@ -0,0 +1,253 @@
+
+There are 2 methods for this part. I'll do **RPKG** first since it's a bit more easier to find and confirm your image directly. I still suggest you try the **GlacierKit** method too for reasons I will explain later on.
+
+
+### Method 1: Using RPKG
+
+1. Open **RPKG** 
+2. In **RPKG**, click on *Import* in the top left corner.
+3. Select *Import RPKGs folder* and locate your Hitman 3 runtimes folder (it usually looks like this: `D:\SteamLibrary\steamapps\common\HITMAN 3\Runtime`).
+4. Once the folder is imported, click on the big **REPO** tab button next to *Resource Overview*.
+5. Then, click *Load REPO from RPKGS folder*. Now you’ll see everything neatly ordered with a good overview of each category. This is the **Repository**.
+6. Expand the *Outfits* section below. Search for your outfit there. Or use the search bar on top.
+
+>Leave your **RPKG** program open after finding your suit
+
+### Method 2: Using GlacierKit
+
+1. Open **GlacierKit**
+2. Select the *game contents* tab on the left.
+3. - Copy the following **repository's** file hash  ```00204D1AFD76AB13``` and paste it on the search bar there and hit enter. 
+   - Or simply type ***pro.repo*** on the search bar and hit enter.
+Both methods should bring you to a single file called ***pro.repo***. 
+4. Open it by clicking on it (obviously) and then click on *Open in editor* in the new menu that popped up on the right. This is the **repository** but in **Glacierkit**
+
+:::tip
+You now know the exact name of the **repository** file. Next time you want to open the **repository** in **GlacierKit** just type ***pro.repo***.
+::: 
+
+5. You can then start searching for your suit below in the *Unmodified section*.
+6. Double check if you have the right suit selected by looking up the image in **GlacierKit**.
+To do so, simply 
+
+   - find the correct image code line on the right that starts with ``"Image":".....``
+ 
+   - and locate its **assembly path** 
+
+
+
+>The **assembly path** is the line after ``"Image":"..."`` inside the quotation marks.
+
+     Example of a full image code: 
+   ```json
+   "Image": "images/unlockables_override/47_outfits_bloodmoney_gloves.jpg"
+   ```
+   and its **assembly path**
+   ```json
+   images/unlockables_override/47_outfits_bloodmoney_gloves.jpg
+   ```
+
+- Copy it and paste the **assembly path** in the search bar on top in the *game contents* tab on the left 
+- and hit enter on your keyboard.
+
+
+If the image you found matches the suit you want to modify, you can go back to your **repository** file. (Where you saw the **repository** code).
+
+>Leave open your **GlacierKit** program
+
+:::info GlacierKit vs. RPKG
+While **RPKG** is helpful for viewing images in the **repository**, it’s best to work primarily in **GlacierKit**, as it will be your main modding tool. The **repository code lines** in both tools are identical for each file, so practicing in **GlacierKit** is encouraged for efficient modding.
+:::
```

Comment:
I do think it'll be a little confusing to show two separate tools for doing something and then say "it's best to not do the first one we told you to do"